flow_create_readme/writeInfo-genePred.cwl

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: [bash, -c] 
+requirements:
+  - class: InlineJavascriptRequirement
+  - class: InitialWorkDirRequirement
+    listing:
+      ${
+        var LIST = [(inputs.readme_file)];
+        return LIST;
+      }
+   
+inputs:
+  in_dummy:
+    type: File[]
+  readme_file:
+    type: File
+  tree:
+    type: string[]
+  link_to_publication:
+    type: string
+
+   
+arguments:
+  - position: 1
+    valueFrom: "echo -e '\nInformation about this dataset can be found here: https://i5k.nal.usda.gov/content/ncbi-$(inputs.tree[2])\nInformation about the publication of this dataset can be found here: $(inputs.link_to_publication)' >> readme.txt"
```

Comment:
Hmm, looks like we'll need to modify $(inputs.tree[2]) - for the URL to work, this needs to be 1) all lowercase, and 2) the underscores `_` need to be replaced with hyphens `-`. E.g. https://i5k.nal.usda.gov/content/ncbi-apis-mellifera-annotation-release-103

Reply:
Okay! I can change this by using javascript.